### DISCUSSION TITLE
Deployable package on AWS

### DISCUSSION THREAD

**Discussion by ragaconsulting** [deepgram/community#983] [Feature Requests]
Hi Team,

Does anyone have a deployable speech-to-text test app that leverages AWS and Deepgram? Ideally, it would be serverless, using Lambda and API Gateway WebSockets. I’d like to run a quick POC with our client, and I’m aiming to keep development work minimal.

Thanks in advance!

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment]
Hi @ragaconsulting, check out our Deepgram SDK examples for [JavaScript](https://github.com/deepgram/deepgram-js-sdk/tree/main/examples), [Python](https://github.com/deepgram/deepgram-python-sdk/tree/main/examples), [Go](https://github.com/deepgram/deepgram-go-sdk/tree/main/examples), and [.NET/C#](https://github.com/deepgram/deepgram-dotnet-sdk/tree/main/examples).

These aren't written specifically for AWS, but they're out-of-the-box examples of both pre-recorded and streaming transcription. They should be a great starting point that you could run locally or incorporate into a Lambda function.
